lib/content-services/src/lib/search/components/search-check-list/search-check-list.component.ts

```diff
@@ -40,6 +40,7 @@ export interface SearchListOption {
 export class SearchCheckListComponent implements SearchWidget, OnInit {
 
     id: string;
+    name: string;
```

Comment:
you are adding non-nullable property that can be considered a breaking change... have you tested this in ADW that everything works fine?